crawl-ref/source/dat/des/variable/float.des

```diff
@@ -1411,6 +1411,22 @@ MAP
      ...                             ...
 ENDMAP
 
+NAME:   crystal_passage
+DEPTH:  D, Depths
+TAGS:   transparent extra no_monster_gen
+KITEM:   d = any potion
+MAP
+xxxb@bxxx
+xxxb.bxxx
+xxxb.bxxx
+bbbT.Tbbb
```

Comment:
You can see the rock walls in the corners behind the fountains on crystal_passage. I'm not sure if this is intended; it's a very easy fix if not. Don't forget to credit yourself in the vault name!